src/util/artifacts.js

```diff
@@ -12,7 +12,7 @@ export async function prepareArtifactsDir(
   try {
     const stats = await fs.stat(artifactsDir);
     if (!stats.isDirectory()) {
-      throw new WebExtError(`${artifactsDir} is not a directory`);
+      throw new UsageError(`${artifactsDir} is not a directory`);
```

Comment:
It would be easier for developers to understand this if you change the message to: `--artifacts-dir=${artifactsDir} (this value is not a directory)`